lmms_eval/tasks/camerabench_vqa/utils.py

```
@@ -0,0 +1,290 @@
+import datetime
```

Ruff (0.11.9)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)